app/src/pages/inside/launchesPage/modals/launchFinishForceModal/launchFinishForceModal.jsx

```diff
@@ -0,0 +1,81 @@
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { withModal, ModalLayout } from 'components/main/modal';
+import { userIdSelector } from 'controllers/user';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import styles from './launchFinishForceModal.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  finishForceLaunchHeader: {
+    id: 'LaunchFinishForceDialog.finishForceLaunchHeader',
+    defaultMessage: 'Force finish',
+  },
+  finishForceLaunchText: {
+    id: 'LaunchFinishForceDialog.finishForceLaunchText',
+    defaultMessage: 'Are you sure you want to finish the launch? All data will be missed.',
+  },
+  finishForceLaunchWarning: {
+    id: 'LaunchFinishForceDialog.finishForceLaunchWarning',
+    defaultMessage:
+      'You are going to stop not your own launch. This may affect other users information on the project.',
+  },
+});
+
+@withModal('launchFinishForceModal')
+@injectIntl
+@connect((state) => ({
+  userId: userIdSelector(state),
+}))
+export class LaunchFinishForceModal extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    data: PropTypes.shape({
+      item: PropTypes.object,
+      onConfirm: PropTypes.func,
+    }),
+    userId: PropTypes.string.isRequired,
+  };
+
+  static defaultProps = {
+    data: {
+      item: {},
+      onConfirm: () => {},
+    },
+  };
+  confirmAndClose = (closeModal) => {
+    this.props.data.onConfirm();
+    closeModal();
+  };
+
+  render() {
+    const { intl } = this.props;
+    const { item } = this.props.data;
+    const okButton = {
+      text: intl.formatMessage(COMMON_LOCALE_KEYS.FINISH),
+      danger: true,
+      onClick: this.confirmAndClose,
+    };
+    const cancelButton = {
+      text: intl.formatMessage(COMMON_LOCALE_KEYS.CANCEL),
+    };
+    return (
+      <ModalLayout
+        title={intl.formatMessage(messages.finishForceLaunchHeader)}
+        okButton={okButton}
+        cancelButton={cancelButton}
+        warningMessage={
```

Comment:
(item.owner !== this.props.userId) && intl.formatMessage(messages.finishForceLaunchWarning)

Reply:
fixed